dbt/tests/data_tests/singular_tests/validate_out_gridpathratoolkit__hourly_available_capacity_factor__aggregation_groups.sql

```diff
@@ -0,0 +1,17 @@
+-- Check that every capacity factor aggregation key appears in the aggregations.
+--
+-- This isn't a normal foreign-key relationship, since the aggregation group isn't the
+-- primary key in the aggregation tables, and is not unique in either of these tables,
+-- but if an aggregation group appears in the capacity factor time series and never
+-- appears in the aggregation table, then something is wrong.
```

Comment:
Should this live in the `schema.yml` file?

Reply:
It's a singular test, so there is no schema.yml file...

Reply:
mmmm ok I see